tee-worker/scripts/litentry/release/deploy.sh

```diff
@@ -189,14 +213,24 @@ function restart_parachain() {
   local service_name="relay-alice"
   local description="Alice Node for Relay Chain"
   local working_directory="$TMPDIR"
-  local log_file=/tmp/parachain_dev/relay.alice.log
+  if [ "$ROOT" = true ]; then 
+    local log_file=/opt/parachain_dev/relay.alice.log
+  else 
+    local log_file=/tmp/parachain_dev/relay.alice.log
+  fi 
   local command="$POLKADOT_BIN --chain $ROCOCO_CHAINSPEC --alice --tmp --port ${AlicePort:-30336} --ws-port ${AliceWSPort:-9946} --rpc-port ${AliceRPCPort:-9936}"
```

Comment:
I think we have a hidden problem here:

we shouldn't use `--tmp`. We should use `--base-path` to persist the storage: imagine we want to restart the parachain (or the server simply reboots)

This applies to relaychains + parachain

Reply:
I see, I was using the same commands as `launch-local-binary.sh`, I will update and test this, but I have another questions, if it persists the data, do you have an idea where it stores it, and does it also mean if we restart with the persisted data we don't have to `regsiter_parachain` again?

Reply:
> and does it also mean if we restart with the persisted data we don't have to regsiter_parachain again?

yes.

the stored data is exactly under `--base-path`
We need 3 different folders for 3 dbs: 2 relaychains + 1 parachain

**We should test** it (e.g. restarting the parachain service) after it's deployed to make sure the data is actually persisted.

Reply:
I like this, This makes the system to be able to get back on server reboots and the data persists, so we don't have to call any scripts. It will take me some time to test this, but this should be a good improvement.

Reply:
Thanks! We have to be time conscious too - ideally we can get it done today